pilot/pkg/serviceregistry/kube/controller/endpointcontroller.go

```diff
@@ -53,16 +53,14 @@ func (e *kubeEndpoints) Run(stopCh <-chan struct{}) {
 	e.informer.Run(stopCh)
 }
 
-// handleEvent processes the event.
-func (e *kubeEndpoints) handleEvent(name string, namespace string, event model.Event, ep interface{}, fn updateEdsFunc) error {
+// triggerConfigUpdate triggers the config update.
+func triggerConfigUpdate(c *Controller, epc kubeEndpointsController, name string, namespace string, event model.Event, ep interface{}) error {
```

Comment:
nit: config update to me implies an Istio CR, not k8s registry info

Reply:
Changed it to `processEndpointEvent`